src/include/Rinternals.h

```diff
@@ -1392,9 +1434,12 @@ void R_orderVector1(int *indx, int n, SEXP x,       Rboolean nalast, Rboolean de
    with or without the Rf_ prefix.
 */
 SEXP     Rf_allocVector(SEXPTYPE, R_xlen_t);
-Rboolean Rf_conformable(SEXP, SEXP);
-SEXP	 Rf_elt(SEXP, int);
+//@MOVED: Rf_conformable unused
+//Rboolean Rf_conformable(SEXP, SEXP);
+//@MOVED: Rf_elt unused
+//SEXP	 Rf_elt(SEXP, int);
 Rboolean Rf_inherits(SEXP, const char *);
+// The following functions seem too fundamental to be moved, although some of them are not used
```

Comment:
The following functions seem too fundamental to be moved, although some of them are not used